modules/tinymce/src/themes/silver/main/ts/ui/context/ContextToolbarScopes.ts

```diff
@@ -25,7 +26,7 @@ const categorise = (contextToolbars: Record<string, Toolbar.ContextFormApi | Too
   const inNodeScope: Array<Toolbar.ContextForm | Toolbar.ContextToolbar> = [ ];
   const inEditorScope: Array<Toolbar.ContextForm | Toolbar.ContextToolbar> = [ ];
```

Comment:
Should these be using the new `ContextTypes` type?

Reply:
Ah, yep. Thanks!